components/UI/iconsComponents/icons/eyeIcon.tsx

```diff
@@ -0,0 +1,11 @@
+
+const EyeIcon = () => {
```

Comment:
Here please follow the same pattern as for the other icons:
<img width="957" alt="image" src="https://github.com/user-attachments/assets/a87f91fe-64ea-44c4-9fa5-7f36149ff3ac">
(Missing types and imports)